me/prunt/simplejail/Main.java

```diff
@@ -39,6 +45,44 @@ public void onEnable() {
         // Creates scheduler to unjail players when necessary every 1 minute
         createUnjailScheduler();
     }
+    @EventHandler
+    public void onVehicleEnter(VehicleEnterEvent e){
+        if(e.getEntered() instanceof Player player){
+            if(isJailed(player)){
+                player.sendMessage(getMessage("messages.cant-enter-in-vehicle"));
+                e.setCancelled(true);
+            }
+        }
+    }
+    @EventHandler
```

Comment:
Please respect already existing formatting and add empty lines where reasonable.